migrations/generic/timetables/1676027954829_add_hastus_fields/up.sql

```diff
@@ -0,0 +1,52 @@
+CREATE TABLE vehicle_journey.journey_type (
+  type text PRIMARY KEY
+);
+COMMENT ON TABLE vehicle_journey.journey_type IS 'Enum table for defining allowed journey types.';
+
+INSERT INTO vehicle_journey.journey_type
+  (type)
+  VALUES
+  ('STANDARD'),
+  ('DRY_RUN'),
+  ('SERVICE_JOURNEY')
+  ON CONFLICT (type) DO NOTHING;
+
+ALTER TABLE vehicle_journey.vehicle_journey
+  ADD COLUMN journey_name_i18n jsonb,
+  ADD COLUMN turnaround_time interval,
+  ADD COLUMN layover_time interval,
+  ADD COLUMN journey_type text NOT NULL DEFAULT 'STANDARD' REFERENCES vehicle_journey.journey_type (type);
+COMMENT ON COLUMN vehicle_journey.vehicle_journey.journey_name_i18n IS 'Name that user can give to the vehicle journey.';
+COMMENT ON COLUMN vehicle_journey.vehicle_journey.turnaround_time IS 'Turnaround time. Means the time it takes to move vehicle to the starting stop and pick up the passengers.';
+COMMENT ON COLUMN vehicle_journey.vehicle_journey.layover_time IS 'Layover time. 95% of the journeys should arrive to the terminal within journey time + recovery time so that they would be ready to start next journey in time.';
```

Comment:
The Transmodel definition might be a bit better: LAYOVER TIMEs describe a certain time allowance that may be given at the end of each VEHICLE JOURNEY, before starting the next one, to compensate delays or for other purposes (e.g. rest time for the driver). This “layover time” can be regarded as a buffer time, which may or may not be actually consumed in real time operation.